src/item/ItemCooldownTags.php

```diff
@@ -27,12 +27,17 @@
  * Tags used by items to determine their cooldown group.
  * @see Item::getCooldownTag()
  */
-final class VanillaCooldownTags{
+final class ItemCooldownTags{
 
 	private function __construct(){
 		//NOOP
 	}
 
+	/**
+	 * These tag values are not related to Minecraft internal IDs.
+	 * They only share a visual similarity because these are the most obvious values to use.
+	 * Any arbitrary string can be used.
+	 */
```

Comment:
This comment would be better placed in the class docblock so it shows up in the correct place on doc.pmmp.io